HarpGuru/HarpFace/types/index.ts

```diff
@@ -1,7 +1,10 @@
 import type { HarpStrata } from 'harpstrata'
 
+import type { HarpRowProps } from '../HarpRow'
 import type { SetActiveHarpStrata } from '../../HarpGuru'
 
+// TODO: this should probably be moved in to HarpCell since
+// that's where it's really required
```

Comment:
Link to this comment in https://github.com/js-jslog/harpnative/issues/31 and then delete this TODO

Reply:
Done